src/libs/state.ts

```diff
@@ -6,17 +6,27 @@ import type { WorkspaceWrapper } from 'src/libs/workspace-utils';
 export const routeHandlersStore = Utils.atom<unknown[]>([]);
 
 export const authStore = Utils.atom<any>({
-  isSignedIn: undefined,
   anonymousId: undefined,
```

Comment:
It's great to have everything that this store might contain listed out here. +1 for sorting the keys.